databroker/headersource/base.py

```diff
@@ -1,8 +1,16 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 from . import core
+import numpy as np
 
 
+def _sanitize_np(val):
```

Comment:
In #195 (awaiting fixes before merge) I add `sanitize_np` a public function in utils. Leaving this here is fine for this PR, but removing the leading underscore from the name would make the rebase easier. Optional!